lib/asciidoctor/stylesheets.rb

```diff
@@ -0,0 +1,106 @@
+# encoding: UTF-8
+# Monkey patch the whole file because the class is static
```

Comment:
@mojavelinux  Wow thanks for taking the time to dig into this issue!

> Of course, another solution to the problem is to literally patch the JS file as we do for the gsub stuff. The could be a cleaner approach.

Yeah, I will patch the JS file, that's the less intrusive solution and maybe the most comprehensive.